core/src/main/resources/lib/hudson/rssBar.jelly

```diff
@@ -24,22 +24,22 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:x="jelly:xml" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <div align="right" style="margin:1em">
-    <a href="${rootURL}/legend">${%Legend}</a>
-    <span style="padding-left:1em">
-      <a href="rssAll"><img src="${imagesURL}/atom.gif" border="0" alt="Feed" height="16" width="16"/></a>
+  <div id="rss-bar">
+    <a href="${rootURL}/legend" id="rss-bar-legend-link">${%Legend}</a>
+    <span id="rss-bar-rss-all" class="rss-bar-item">
+      <a href="rssAll" class="rss-bar-item-icon-link"><img src="${imagesURL}/atom.gif" class="icon-rss icon-sm" alt="Feed"/></a>
       <st:nbsp/>
-      <a href="rssAll">RSS ${%for all}</a>
+      <a href="rssAll" class="rss-bar-item-text-link">RSS ${%for all}</a>
     </span>
-    <span style="padding-left:1em">
-      <a href="rssFailed"><img src="${imagesURL}/atom.gif" border="0" alt="Feed" height="16" width="16"/></a>
+    <span id="rss-bar-rss-failed" class="rss-bar-item">
+      <a href="rssFailed" class="rss-bar-item-icon-link"><img src="${imagesURL}/atom.gif" class="icon-rss icon-sm" alt="Feed"/></a>
       <st:nbsp/>
-      <a href="rssFailed">RSS ${%for failures}</a>
+      <a href="rssFailed" class="rss-bar-item-text-link">RSS ${%for failures}</a>
     </span>
-    <span style="padding-left:1em">
-      <a href="rssLatest"><img src="${imagesURL}/atom.gif" border="0" alt="Feed" height="16" width="16"/></a>
+    <span id="rss-bar-rss-latest" class="rss-bar-item">
```

Comment:
If the code would look like this:

```html
    <span class="rss-bar-item">
      <a href="rssAll" class="rss-bar-item-icon-link"><img src="${imagesURL}/atom.gif" class="icon-rss icon-sm" alt="Feed"/></a>
      <st:nbsp/>
      <a href="rssAll" class="rss-bar-item-text-link">RSS ${%for all}</a>
    </span>
    <span class="rss-bar-item">
      <a href="rssFailed" class="rss-bar-item-icon-link"><img src="${imagesURL}/atom.gif" class="icon-rss icon-sm" alt="Feed"/></a>
      <st:nbsp/>
      <a href="rssFailed" class="rss-bar-item-text-link">RSS ${%for failures}</a>
    </span>
    <span class="rss-bar-item">
      <a href="rssLatest" class="rss-bar-item-icon-link" ><img src="${imagesURL}/atom.gif" class="icon-rss icon-sm" alt="Feed"/></a>
      <st:nbsp/>
      <a href="rssLatest" class="rss-bar-item-text-link">RSS ${%for just latest builds}</a>
    </span>
```


How would I identify the img of the rssFailed?
Are you suggesting to use CSS:

```css
#rss-bar-item a[href$=rssFailed] img {
   border:2px solid red;
}
```

Because then the UI breaks if someone changes the href to something else.
That's why I like to have ids, but if you are persistent, I can remove the IDs and
use - the in my optinion ugly - pseudo selectors in my external css.